src/Advisor/Advisor/Cmdlets/Utilities/RecommendationHelper.cs

```diff
@@ -51,8 +51,8 @@ public static List<PsAzureAdvisorResourceRecommendationBase> RecommendationFilte
 
             List<PsAzureAdvisorResourceRecommendationBase> filteredList = new List<PsAzureAdvisorResourceRecommendationBase>();
 
-            // Filter by category only if its a valid input
-            if (Category.Cost.Equals(category) || Category.HighAvailability.Equals(category) || Category.Performance.Equals(category) || Category.Security.Equals(category))
+            // Filter by category
+            if (!string.IsNullOrEmpty(category))
```

Comment:
the change breaks not test?

Reply:
It was redundant check. Input already checks valid category names. Simplified it.